project/CaseClass.scala

```diff
@@ -6,7 +6,11 @@ case class CaseClass(
     val name = Sanitize(this.name)
     s"""|final case class $name(
         |${parameters.map("  " + _).mkString(",\n")}
-        |) derives _root_.io.circe.Codec.AsObject
+        |)
+        |
+        |object $name {
+        |  implicit val ${name}Codec: _root_.io.circe.Codec[$name] = _root_.io.circe.generic.semiauto.deriveCodec
+        |}
```

Comment:
Yeah, I guess so! (along with the change in wildcard import syntax)

Reply:
Cool! So besides cross-publishing core, would it make sense to publish the `DIscoveryPlugin` too maybe? Or do you want to PR Google Drive support here?

Reply:
It would be nice to have Drive support in one place, but either way works for me. We can always merge Drive support later on, if it starts off being developed in a different repo.

Reply:
Drive support here is perfect :) I just couldn't tell how serious you were about it from your comment!

> but if we wanted to actually include Drive, I would go figure out how those values actually need to be encoded/decoded.